### ISSUE TITLE
Bug. Dashboard EMBEDDED. DOWNLOAD XLS/CSV with more than 10000 rows, Shows only 10000 rows. Only 2000 if there is RawData (not view agrupation like a count)

### ISSUE THREAD

**Issue by jlisyc** [metabase/metabase#9651]
### Bug. Dashboard EMBEDDED. DOWNLOAD XLS/CSV with more than 10000 rows, Shows only 10000 rows. Only 2000 if there is RawData (not view agrupation like a count)

- Your browser and the version: (Firefox 60.0.1,)
- Your operating system: (ubuntu 16.04)
- Your databases: (mysql, h2, postgres,...)
- Metabase version: (0.31.2)
- Metabase hosting environment: (Linux/Ubuntu)
- Metabase internal database: (Postgres)

## Steps to reproduce
1. H2 Exampe. ORDERS
![image](https://user-images.githubusercontent.com/33416503/55338241-90105200-54a0-11e9-9906-e4e72d6673da.png)

2. Create dashboard based in that question and Embedd it.

4. try to download in embedd space. You can see it shows only 10000.
![image](https://user-images.githubusercontent.com/33416503/55340113-52152d00-54a4-11e9-904b-110c58c11778.png)

-------------------
Thanks for Metabase project!
-------

**Comment by flamber** [comment]
Confirmed on 0.32.1, but only when doing embedding via dashboard - when embedding via question, it downloads correctly.

Embedded dashboard request:
```
/api/embed/dashboard/...UUID.../dashcard/...IDX.../card/...IDY.../csv
```
Embedded question request:
```
/api/embed/card/...UUID.../query/csv
```

@jlisyc The open issues you've referenced are not related to this issue. They are specific to BigQuery.

**Comment by jlisyc** [comment]
@flamber Thanks a lot. 
I thought was related.
For us is important to be able to download all rows in embedded dashboards.

**Comment by jlisyc** [comment]
I have embedded also a question
![image](https://user-images.githubusercontent.com/33416503/55393800-36f6fb80-553e-11e9-8488-1ea1bde262b6.png)

and when a try to download , the question iframe disappear

**Comment by flamber** [comment]
Okay, that sounds like a completely different issue. You need to check Metabase logs and browser console for any errors.

**Comment by jlisyc** [comment]
Related https://github.com/metabase/metabase/issues/9831

**Comment by camsaul** [comment]
Fixed by #9872
